_layouts/post.html

```diff
@@ -0,0 +1,147 @@
+---
+layout: default
+---
+
+<!--================Forum Breadcrumb Area =================-->
+<section class="doc_banner_area search-banner-light" style='padding:50px 0'>
+</section>
+<section class="page_breadcrumb">
+  <div class="container custom_container">
+    <div class="row">
+      <div class="col-sm-7">
+        <nav aria-label="breadcrumb">
+          <ol class="breadcrumb">
+            <li class="breadcrumb-item"><a href="https://www.lovelace.academy">Home</a></li>
+            <li class="breadcrumb-item"><a href="https://learn.lovelace.academy">{{ page.categories[0] }}</a></li>
+            <li class="breadcrumb-item active" aria-current="page">{{ page.title }}</li>
+          </ol>
+        </nav>
+      </div>
+      <div class="col-sm-5">
+        <a href="#" class="date"><i class="icon_clock_alt"></i>Updated on {{ page.last_modified_at | default: page.date | date: '%b %d, %y' }}</a>
+      </div>
+    </div>
+  </div>
+</section>
+<!--================End Forum Breadcrumb Area =================-->
+
+<!--================Topic Area =================-->
+<section class="doc_documentation_area" id="sticky_doc">
+  <div class="overlay_bg"></div>
+  <div class="container custom_container">
+    <div class="row">
+      <div class="col-lg-3 doc_mobile_menu display_none">
+        <aside class="doc_left_sidebarlist">
+          <div class="open_icon" id="left">
+            <i class="arrow_carrot-right"></i>
+            <i class="arrow_carrot-left"></i>
+          </div>
+          <div class="scroll">
+            <ul class="list-unstyled nav-sidebar">
+              <li class="nav-item">
+                <a href="" class="nav-link"><img src="/img/side-nav/home.png" alt="">Home</a>
+              </li>
+              <li class="nav-item active">
+                <a href="#" class="nav-link"><img src="/img/side-nav/briefcase.png" alt="briefcase">Getting Started</a>
+                <span class="icon"><i class="arrow_carrot-down"></i></span>
+                <ul class="list-unstyled dropdown_nav">
+                  <li><a href="/why-cardano/">Why Build On Cardano</a></li>
```

Comment:
Not fully sure why but the order of this seems off on the deployed test site e.g. https://fauno.github.io/Learn/keys-and-addresses

Reply:
I gave it a manual order at #2